src/main/java/net/pms/util/OpenSubtitle.java

```diff
@@ -1264,30 +1116,19 @@ protected static String findImdbIdByFileHash(
 		return searchSubtitles(queryArray, resource, prettifier, "filename", fileName, -1);
 	}
 
-	private static void addGuesses(
-		List<GuessCandidate> candidates,
-		Collection<? extends GuessItem> guesses,
-		FileNamePrettifier prettifier,
-		VideoClassification classification,
-		Locale locale
-	) {
-		// The score calculation isn't extensively tested and might need to be tweaked
-		for (GuessItem guess : guesses) { //TODO: (Nad) Guess Scoring
+	private static void addGuesses(List<GuessCandidate> candidates, Collection<? extends GuessItem> guesses, FileNamePrettifier prettifier,
+		VideoClassification classification, Locale locale) {
+		// The score calculation isn't extensively tested and might need to be
+		// tweaked
+		for (GuessItem guess : guesses) { // TODO: (Nad) Guess Scoring
 			double score = 0.0;
 			if (isBlank(prettifier.getName()) || isBlank(guess.getTitle())) {
 				continue;
 			}
-			score += new JaroWinklerSimilarity().apply(
-				prettifier.getName().toLowerCase(locale),
-				guess.getTitle().toLowerCase(locale)
-			);
-			if (score < MIN_IMDB_GUESS_JW_DISTANCE) { //TODO: (Nad) Parameterize threshold...?
-				LOGGER.trace(
-					"OpenSubtitles: Excluding IMDB guess because the similarity ({}) is under the threshold ({}): {}",
-					score,
-					MIN_IMDB_GUESS_JW_DISTANCE,
-					guess
-				);
+			score += new JaroWinklerSimilarity().apply(prettifier.getName().toLowerCase(locale), guess.getTitle().toLowerCase(locale));
+			if (score < MIN_IMDB_GUESS_JW_DISTANCE) { // TODO: (Nad) Parameterize threshold...?
+				LOGGER.trace("OpenSubtitles: Excluding IMDB guess because the similarity ({}) is under the threshold ({}): {}", score,
+					MIN_IMDB_GUESS_JW_DISTANCE, guess);
```

Comment:
```suggestion
				LOGGER.trace(
					"OpenSubtitles: Excluding IMDB guess because the similarity ({}) is under the threshold ({}): {}",
					score,
					MIN_IMDB_GUESS_JW_DISTANCE,
					guess
				);
```